File: php/ver.php
<?php 
  include("restclient.php"); 
  $api = new RestClient(array(
     'base_url' => "localhost:8080/api/")
  );
  $result = $api->get("usuarios/".$_GET['id']);
  $json_data = json_decode($result->response);
 ?>

<link rel="stylesheet" type="text/css" href="./static/css/bootstrap.min.css">

  <a href="index.php">Indice</a>

  <table class="table table-bordered">
  <th></th>
  <th>Email</th>
  <th>Nombres</th>
  <th>Apellidos</th>
  <tr>
    <td><a href="editar.php?id=<?php echo $_GET['id'] //$listado->_id ?>" class=" fa fa-pencil">Editar </a></td>
    <td><?php echo $json_data->email?></td>
    <td><?php echo $json_data->nombres?></td>
    <td><?php echo $json_data->apellidos?></td>
  </tr>
  </table>

File: perl/ver.pl
#!/usr/bin/perl
use LWP::Simple;
use JSON::PP;
use CGI;
my $query = new CGI;
my $user_id = $query->param('user_id');

$usuario= decode_json(get("http://localhost:8080/api/usuarios"."/".$user_id));

print "Content-type: text/html\n
<html>
  <head>
    <title>Editar usuario</title>
  </head>\n
  <body>
	  <table class='table table-bordered'>
	  	<th>Email</th>
	  	<th>Nombres</th>
	  	<th>Apellidos</th>
	    <tr>
 		  <td> $usuario->{email}</td>
 		  <td> $usuario->{nombres} </td>
 		  <td> $usuario->{apellidos}</td>
	    </tr>
      </table>
      <a href='/editar.pl?user_id=$usuario->{_id}'>Actualizar Usuario</a> 
  </body>
</html>";
